static/js/src/dynamic-forms.js

```diff
@@ -408,14 +408,15 @@ import { prepareInputFields } from "./prepare-form-inputs.js";
             message += "\r\n\r\n";
           });
 
-          const howManyMachinesFieldset =
-            document.getElementById("how-many-machines");
-          const machinesInputs = howManyMachinesFieldset?.querySelectorAll(
-            "input[name='how-many-machines-do-you-have']",
-          );
-          machinesInputs.forEach((input) => {
-            input.removeAttribute("name");
-          });
+          const radioFieldsets = document.querySelectorAll(".js-remove-radio-names");
```

Comment:
Instead of checking for the specific `"input[name='how-many-machines-do-you-have']"`, we will check for fieldsets with the `js-remove-radio-names` classname and remove the radio name attributes accordingly. Not all radio fieldsets need to be removed, only the ones that we would like to include in the payload such as `Comments_from_lead_C`